Misc/NEWS.d/next/Windows/2021-05-02-15-29-33.bpo-43984.U92jiv.rst

```diff
@@ -0,0 +1,3 @@
+:meth:`winreg.SetValueEx` now leaves the target register untouched in case of conversion errors.
+Previously, ``-1`` would be written in case of such errors.
+
```

Comment:
It should be noted that this change only affects `REG_DWORD` and `REG_QWORD` values.

Reply:
No problem. But let me add the tests first.

Reply:
As a point of English: "in case of" here doesn't mean what you want it to. "in case of ..." means: it leaves it untouched in every situation, and the reason is that conversion errors might happen. What you want is "in the case of ..." which means: if a conversion error happens, the value is left untouched.

Reply:
Thanks for pointing that out, @tjguk! It was my suggestion, but I'm no native English speaker :)